tests/smoke_test_dse/smoke_test_dse_suite_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package smoke_test_dse
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ginkgo_util "github.com/datastax/cass-operator/mage/ginkgo"
+	"github.com/datastax/cass-operator/mage/kubectl"
+)
+
+var (
+	testName     = "Cluster resource cleanup after termination"
```

Comment:
Maybe update the testName to be "Smoke test of basic functionality for one-node DSE cluster."

Reply:
Ah good catch on these... I hadn't updated them from the termination test that they were based on